addons/new-osio-addon/pom.xml

```diff
@@ -40,5 +45,20 @@
       <artifactId>hoverfly-java</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.fabric8.launcher</groupId>
+      <artifactId>launcher-core-impl</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.fabric8.launcher</groupId>
+      <artifactId>launcher-service-openshift-impl</artifactId>
+      <scope>test</scope>
```

Comment:
Aren't these services transitive dependencies in core-impl?

Reply:
only git is, changed in #84